appveyor.yml

```diff
@@ -18,7 +18,7 @@ before_build:
             $env:TARGET="VC-WIN32 no-asm"
         } Else {
             $env:VCVARS_PLATFORM="amd64"
-            $env:TARGET="VC-WIN64A-masm"
+            $env:TARGET="VC-WIN64A-masm --strict-warnings"
```

Comment:
Hmm....I thought 32-bit builds were clean and 64-bit ones are not at the moment? Or am I wrong and its the other way around?

Reply:
You're entirely correct.  Will fix

Reply:
Ah...10-main.conf currently adds "/WX" for VC-WIN32 builds. So probably appveyor should have "--strict-warnings" for VC-WIN32 builds, and then remove that flag from 10-main.conf.

Reply:
Yeah, that's what I ended up doing